### DISCUSSION TITLE
`favicons.html` and quote nesting

### DISCUSSION THREAD

**Discussion by jamesbraza** [thegeeklab/hugo-geekdoc#640] [Ideas]
https://geekdocs.de/features/theming/#full-replacement

```html
<link rel="icon" type="image/svg+xml" href="{{ "favicon/favicon.svg" | relURL }}">
```

This works in practice, though it also renders as an error in PyCharm:

![image](https://github.com/thegeeklab/hugo-geekdoc/assets/8990777/7a37e920-9389-417e-9edd-cfddc2d25075)

Nesting with double quotes on the outside, single quotes on the inside leads to a parser error:

```html
 process: "/path/to/site/layouts/partials/head/favicons.html:1:1": parse failed malformed character constant: 'favicon/favicon.ico'
```

Nesting with single quotes on the outside, double quotes on the inside both works and doesn't give an IDE error. Like so:

```html
<link rel="icon" type="image/svg+xml" href='{{ "favicon/favicon.svg" | relURL }}'>
```

Is this something worth incorporating?

**Comment by xoxys** [comment]
This is not a plain HTML file, so plain HTML syntax detection will lead to issues. Personally, what counts for me is the rendered HTML. Introducing a mix of single/double quotes for HTML attributes to make the IDE happy is IMO not worth it.

**Reply by jamesbraza** [reply]
Sounds good, thank you!
